jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -7,49 +7,76 @@ data:
       profiles:
       - name: globals
         env:
-          S3_ENDPOINT_URL: $(s3_endpoint_url)
+          - name: S3_ENDPOINT_URL
+            value: $(s3_endpoint_url)
         resources:
-          mem_limit: 2Gi
-          cpu_limit: 1
+          requests:
+            memory: 1Gi
+            cpu: 500m
+          limits:
+            memory: 2Gi
+            cpu: 1
 
       - name: Spark Notebook
         images:
         - 's2i-spark-minimal-notebook:3.6'
         - 's2i-spark-scipy-notebook:3.6'
+        - 's2i-spark-minimal-notebook:py36-spark2.4.5-hadoop2.7.3-1'
```

Comment:
You can use `s2i-spark-minimal-notebook:py36-spark2.4.5-hadoop2.7.3` for the tag name since it will function as latest build of spark2.4.5 and hadoop2.7.3. I was using the `-1` as build version number for the notebook to maintain that image hash.

Reply:
Ok